landoui/template_helpers.py

```diff
@@ -174,6 +174,14 @@ def linkify_revision_urls(text: str) -> str:
     return re.sub(search, replace, str(text), flags=re.IGNORECASE)
 
 
+@template_helpers.app_template_filter()
+def linkify_revision_ids(text: str) -> str:
```

Comment:
```suggestion
def linkify_revision_id(text: str) -> str:
```

Reply:
This will work with multiple differential revision IDs. Added a test case to reflect that.

Reply:
I see -- I suppose this could be useful in the future however the way we are using the filter in this PR is by explicitly providing a single revision ID.